BootloaderCommonPkg/Library/IppCrypto2Lib/fips.c

```diff
@@ -376,15 +380,19 @@ RunFipsSelftests ()
             return RETURN_UNSUPPORTED;
         }
     }
-/* SM3 is not supported by FIPS
+/* SM3 is not supported by FIPS.
+ * Here It is used for negative testcase execution.
+ */
+#if FixedPcdGetBool(PcdFipsNegativeTestcase)
```

Comment:
Instead of adding a new PCD, why don't we use existing PCD PcdIppHashLibSupportedMask? e.g.:
#if  FixedPcdGet8(PcdIppHashLibSupportedMask) & IPP_HASHLIB_SM3_256

Reply:
done